wiki/Skin/en.md

```diff
@@ -4,7 +4,29 @@ stub: true
 
 # Skin
 
-*See also: [Skinning](/wiki/Skinning)*\
-*For the dedicated sub-forum for skins, see [Skinning](https://osu.ppy.sh/community/forums/15)*
+A **skin** is a set of graphics (images) and custom sample sets (audio clips) that replace the default ones used by osu!. Many players regularly engage in the use of skins to allow for a more customised user experience, should they dislike the default skin. Almost all of what is visible or audible in osu! can be "skinned." 
 
-A **skin** is a set of graphics (images) and custom sample sets (audio clips) that replace the default ones used by osu!. Many players regularly engage in the use of skins to allow for a more customised user experience, should they dislike the default skin. Almost all of what is visible or audible in osu! can be "skinned."
+## Reasons to Use Skins
+
+- Skins can be used to further optimize a player's gameplay, since most skins do away with showing that a player has hit a note perfectly (to only show the minimum amount of information to a player in-game). 
+- Another gameplay-improving feautre that skins provide (for osu! standrad) is chagning the cursor style and trail. This can be especially helpful if a player finds the default skin cursor intruding into the action.
```

Comment:
I would just remove the entire original bullet point and expand more upon custom UI:
```suggestion
- In addition to gameplay focused changes, skins can also offer custom menus which commonly feature characters from various anime and games, as well as being based around certain design styles, like minimalism or techno.
```

Reply:
> I would just remove the entire original bullet point and expand more upon custom UI:

Great suggestion. Don't know why I hadn't thought of that. I guess I'll be split the "why" section into two sub-sections (Performance and UI)